whisper/README.md

```diff
@@ -19,6 +19,27 @@ Install [`ffmpeg`](https://ffmpeg.org/):
 brew install ffmpeg
 ```
 
+Next, download the Whisper PyTorch checkpoint and convert the weights to MLX format:
+
+```
+# Take the "tiny" model as an example. Note that you can also convert a local PyTorch checkpoint in OpenAI's format.
+python convert.py --torch-name-or-path tiny --mlx-path mlx_models/tiny
+```
+
+To generate a 4-bit quantized model, use ``-q`` for a full list of options:
```

Comment:
-q should have single back ticks.

Also put a period after it and start a new sentence at the `For ...`

Reply:
oops